modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/flat/FlatNPMBundleProcessor.java

```diff
@@ -108,6 +127,13 @@ private String _getResourceContent(
 		return Arrays.asList(dependencies);
 	}
 
+	/**
+	 * Process a dependencies type entry of a package.json file adding them to
```

Comment:
By `dependencies type` I'm referring to `dependencies` and `peerDependencies` entries of the package.json file, which have the same inner structure.